# generated by patch-package 6.4.7 on 2021-07-12 13:17:16
#
# command:
#   npx patch-package idb
#
# declared package:
#   lodash-unified: 1.0.1
#
diff --git a/node_modules/lodash-unified/package.json b/node_modules/lodash-unified/package.json
index 153848e..98a26e6 100644
--- a/node_modules/lodash-unified/package.json
+++ b/node_modules/lodash-unified/package.json
@@ -2,7 +2,7 @@
   "name": "lodash-unified",
   "version": "1.0.1",
   "description": "A union entrypoint of lodash for both ESModule and Commonjs.",
-  "main": "./require.cjs",
+  "main": "./import.js",
   "types": "./type.d.ts",
   "exports": {
     "import": "./import.js",
